frontend/components/events/EventPage.tsx

```
@@ -31,8 +31,8 @@ export default function EventPage({ params }: any) {
         setEventData(event);
         if (event.eventTags && typeof event.eventTags === "object") {
           event.eventTags.map((tagId) => {
```

Biome (2.4.6)

[error] This callback passed to map() iterable method should always return a value.

(lint/suspicious/useIterableCallbackReturn)